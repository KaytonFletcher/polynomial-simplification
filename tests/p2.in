4x^2 + 3x - 4x^2
