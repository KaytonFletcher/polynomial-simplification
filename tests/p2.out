(((4*(x)^2)+(3*x))-(4*(x)^2))
0x^2+3x